app/form_builders/govuk_elements_errors_helper.rb

```diff
@@ -1,4 +1,42 @@
 module GovukElementsErrorsHelper
+  def self.error_summary(object, heading, description, options = {})
```

Comment:
📓 I've had a look at the [gem that contains this code](https://github.com/ministryofjustice/govuk_elements_form_builder/blob/4faddb1a9338a17f5e8a0c6e92c94be3834c8b59/app/helpers/govuk_elements_errors_helper.rb) and it is very limited in the way that the display configurations can be defined. Also, it's not in the context of the form (which is really one of the main issues we're trying to solve here), so when we use something like this to define our form:

```ruby
GovukElementsErrorsHelper.error_summary form, 'There were problems saving this form', ''

form_for form, as: 'alias' do |f|
   ...
end
```

The `as: 'alias'` option is not taken into account for displaying the list of errors, making the links to each error point to the wrong anchor :/

Having looked into the source code of the gem, it doesn't seem that a straight simple fix for it, so for the requirements of this PR, I've chosen to keep those changes within our project only.